examples/basic/database_sql/data.go

```diff
@@ -61,15 +66,19 @@ func getData() (series, seasons, episodes []types.Value) {
 		episodes = append(episodes, episodesData...)
 	}
 
-	return
+	return series, seasons, episodes
 }
 
-func getDataForITCrowd(seriesID string) (series types.Value, seasons, episodes []types.Value) {
```

Comment:
I think this changes not required and harmful
Because if I call `getDataForITCrowd` - I want to know about each of three results. 
Old variant helps to understand. New variant cannot helps

Reply:
OK. I returned previous declaration.
I simple followed by linter's comment.